client-v2/src/selectors/__tests__/formSelectors.spec.ts

```diff
@@ -2,30 +2,28 @@ import cloneDeep from 'lodash/cloneDeep';
 import {
   createSelectFormFieldsForCollectionItem,
   defaultFields,
-  supportingFields
+  supportingFields,
+  emailFieldsToExclude
 } from '../formSelectors';
 import { state, stateWithVideoArticle } from 'fixtures/form';
+import without from 'lodash/without';
 
 describe('Form utils', () => {
-  describe('getFormFieldsForCollectionItem', () => {
+  describe('selectFormFieldsForCollectionItem', () => {
     it("should handle articles that don't exist in the state", () => {
       const selectFormFields = createSelectFormFieldsForCollectionItem();
-      expect(selectFormFields(state as any, 'who-are-you')).toEqual([]);
+      expect(selectFormFields(state, 'who-are-you')).toEqual([]);
```

Comment:
I've boyscouted a bit here and removed the unnecessary `any` -- looks like we now model our state in a complete way 🎉